internal/peermanager/peermanager.go

```diff
@@ -94,11 +94,11 @@ func (pm *PeerManager) Connected(p peer.ID, initialWantHaves []cid.Cid) {
 	// If this is the first connection to the peer
 	if pq.refcnt == 1 {
 		// Inform the peer want manager that there's a new peer
-		pm.pwm.AddPeer(p)
+		pm.pwm.addPeer(p)
 		// Record that the want-haves are being sent to the peer
-		pm.pwm.PrepareSendWants(p, nil, initialWantHaves)
+		_, wantHaves := pm.pwm.prepareSendWants(p, nil, initialWantHaves)
```

Comment:
NOTE: Logic change. Is this correct? It looks equivalent, but just _throwing_ the results away seemed wrong.

Reply:
It's equivalent - I think this way is better because it removes any ambiguity